superset/assets/javascripts/dashboard/components/DashboardViewContainer.jsx

```diff
@@ -0,0 +1,305 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import urlLib from 'url';
+
+import * as dashboardActions from '../actions';
+import * as chartActions from '../../chart/chartAction';
+import Dashboard from './Dashboard';
+import { getExploreUrl } from '../../explore/exploreUtils';
+import { areObjectsEqual } from '../../reduxUtils';
+
+const propTypes = {
+  actions: PropTypes.object,
+  initMessages: PropTypes.array,
+  dashboard: PropTypes.object.isRequired,
+  slices: PropTypes.object,
+  datasources: PropTypes.object,
+  filters: PropTypes.object,
+  refresh: PropTypes.bool,
+  timeout: PropTypes.number,
+  user_id: PropTypes.string,
+  isStarred: PropTypes.bool,
+};
+
+class DashboardViewContainer extends React.PureComponent {
```

Comment:
Why naming `DashboardViewContainer` instead of just `DashboardContainer` ?

Reply:
renamed. we now have `DashboardContainer`. but still use `ExploreViewContainer`. We will have redesign for Explore view very soon.